packages/server/src/api_auth/index.ts

```diff
@@ -6,6 +6,11 @@ import { emailPasswordValidator, emailValidator } from '../validators';
 const rateLimit = require('express-rate-limit');
 
 const router = Router();
+const logoutAllSchema = z.object({
```

Comment:
We can put this validator in the validators folder.

Reply:
I will put it in validator's index.js file